src/pages/Home/_MapPage.scss

```diff
@@ -274,7 +285,7 @@
     }
   }
   .country-svg {
-    fill: #6E7377;
+    fill: #6e7377;
```

Comment:
If this color (and other color) is used globally, I think we should consider setting a global color css class?

Reply:
Yes, might be worth a separate issue where we do that for all of the colors